airflow/bin/cli.py

```diff
@@ -192,6 +192,15 @@ def backfill(args, dag=None):
             ti = TaskInstance(task, args.start_date)
             ti.dry_run()
     else:
+        if args.reset_dagruns:
+            DAG.clear_dags(
+                [dag],
+                start_date=args.start_date,
+                end_date=args.end_date,
+                confirm_prompt=True,
+                include_subdags=False,
+                only_include_backfill_dagruns=True)
```

Comment:
nit: personally I'd go with breaking the line and leaving a trailing comma

Reply:
updated